opus/application/static_media/js/search.js

```diff
@@ -185,19 +185,6 @@ var o_search = {
                 let currentUnitVal = $(`#widget__${slugName} .op-unit-${slugName}`).val();
                 newHash += `&${unitSlugWithId}=${currentUnitVal}`;
             }
-            /*
-            Do not perform normalized api call if:
-            1) Input field is empty OR
-            2) Input value didn't change from the last successful search
-            3) Input value didn't match any ranges names
-            */
-            if (currentValue === "" || currentValue === o_search.slugInputValidValueFromLastSearch[slugWithId] ||
-                !o_search.performInputValidation[slugWithId]) {
-                $(e.target).removeClass("search_input_valid search_input_invalid");
-                $(e.target).removeClass("search_input_invalid_no_focus");
-                $(e.target).addClass("search_input_original");
-                return;
-            }
 
```

Comment:
Since the above piece is removed, only one last minor bug exists:
1. Add a delay, like 3 seconds.
2. Open a range widget, for example, observation duration.
3. In min input, type something invalid, like "9x" and wait until yellow border and grey background show up.
4. Quickly remove "9x", and focus out min input. Sometimes red background will not go away.

The reason is because at step 4, no search will be performed but a normalize input api will be called in "input" event and update the border (line 205-217) after the return of api call. At the same time, "focusout" event will also update the border based on whether the input has a class "search_input_invalid" or "search_input_valid" (line 129-136); however this could happen before the return of normalize input api in "input" event and the input background will have the wrong color in this case. The fix for this is to add the following between line 206 & 207:
`$(e.target).removeClass("search_input_invalid_no_focus");`